includes/caching/class-caching.php

```diff
@@ -1307,7 +1307,7 @@ public function update_database_structure() {
 		if ( self::DB_VERSION !== $version || $this->db_table_relations !== $wpdb->get_var( $query ) ) {
 			include_once ABSPATH . 'wp-admin/includes/upgrade.php';
 
-			if ( version_compare( '2020.1.1', $version, '>' ) ) {
+			if ( version_compare( '2020.1.1', $version, '>' ) && $this->db_table_relations === $wpdb->get_var( $query ) ) {
```

Comment:
:no_entry_sign: **Error**: Use placeholders and $wpdb-&gt;prepare(); found $query (*WordPress.DB.PreparedSQL.NotPrepared*).